spec/operations/eligibilities/determine_full_eligibility_spec.rb

```diff
@@ -35,6 +35,10 @@
 
   # Dwayne is UQHP eligible and eligible for non_magi_reasons
   context 'cms simle test_case_a' do
+    before do
+      allow(Date).to receive(:today).and_return(Date.new(2021, 12, 1))
```

Comment:
why are we hard-coding dates like this?

Reply:
There are specific tests in this file that are meant to be run in the context of the year 2021 to compare FPL levels and others.

Reply:
but it's 2022 now, what's the purpose of keeping these tests?

Reply:
To make sure the code works for any year(retrospective in this case).

Reply:
In what scenario would this test fail? E.g. code change, date change, etc.